core_utils/config.py

```diff
@@ -38,4 +39,8 @@ def get_sqlalchemy_url(cls):
                     pass
         return url
 
+    @classmethod
+    def get_deploy_key(cls):
+        return cls.config.get(GITHUB_SECTION, 'deploy_key')
```

Comment:
not necessarily. the key should always be in the config file and even in the event that the key is not there, the whole process should fail.